src/components/forms/Radio/Radio.tsx

```diff
@@ -36,6 +49,7 @@ export const Radio = ({
       />
       <label className="usa-radio__label" htmlFor={id}>
         {label}
+        {description}
```

Comment:
Same feedback as the checkbox PR: https://github.com/trussworks/react-uswds/pull/1104/files#r612477199

Reply:
changed